controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -235,37 +210,42 @@ def test_delete_volume_succeeds(self):
     def test_get_snapshot_return_correct_value(self):
         snapshot_name = "snapshot"
         snapshot_volume_name = "snapshot_volume"
+        snapshot_volume_wwn = "123456789"
         xcli_snapshot = self._get_single_snapshot_result_mock(snapshot_name, snapshot_volume_name)
         self.mediator.client.cmd.vol_list.return_value = xcli_snapshot
-        res = self.mediator.get_snapshot(snapshot_name)
+        res = self.mediator.get_snapshot(snapshot_volume_wwn, snapshot_name)
         self.assertTrue(res.name == snapshot_name)
-        self.assertTrue(res.volume_name == snapshot_volume_name)
+        self.assertTrue(res.source_volume_id == snapshot_volume_wwn)
 
     def test_get_snapshot_same_name_volume_exists_error(self):
         snapshot_name = "snapshot"
         snapshot_volume_name = ""
+        snapshot_volume_wwn = "123456789"
         xcli_snapshot = self._get_single_snapshot_result_mock(snapshot_name, snapshot_volume_name)
         self.mediator.client.cmd.vol_list.return_value = xcli_snapshot
         with self.assertRaises(array_errors.ExpectedSnapshotButFoundVolumeError):
-            self.mediator.get_snapshot(snapshot_name)
+            self.mediator.get_snapshot(snapshot_volume_wwn, snapshot_name)
 
     def test_get_snapshot_raise_illegal_object_name(self):
         snapshot_name = "snapshot"
+        snapshot_volume_wwn = "123456789"
         self.mediator.client.cmd.vol_list.side_effect = [xcli_errors.IllegalNameForObjectError("", snapshot_name, "")]
         with self.assertRaises(array_errors.IllegalObjectName):
-            self.mediator.get_snapshot(snapshot_name)
+            self.mediator.get_snapshot(snapshot_volume_wwn, snapshot_name)
 
     def test_create_snapshot_succeeds(self):
         snapshot_name = "snapshot"
+        snapshot_volume_id = "123456789"
```

Comment:
it seems we still use more than one term for the same value (`snapshot_volume_wwn` vs. `snapshot_volume_id`).
could you align to one of them at least in your diff?

Reply:
done